test/test_helper.rb

```diff
@@ -4,12 +4,18 @@
 
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
-require 'webmock/minitest'
-require 'support/govuk_content_schema_examples'
 require 'capybara/rails'
 require 'slimmer/test_helpers/govuk_components'
 require 'mocha/mini_test'
 
+Dir[Rails.root.join('test/support/*.rb')].each { |f| require f }
+
+class Minitest::Test
+  def teardown
+    Capybara.current_session.driver.clear_memory_cache
```

Comment:
👍 for this solution to the caching issue. Might be worth putting in a comment to explain that this is because we re-use URLs and capypolterphantom caches pages.